jdaviz/tests/test_app.py

```diff
@@ -1,142 +1,154 @@
-import pytest

```

Comment:
what changes were made to `test_app.py` - is it possible to fix the diff so that the whole file doesn't appear as replaced?

Reply:
Oh that's strange. The only change I made was adding that last test at the bottom:

```
def test_vue_remove_data(specviz_helper, spectrum1d):
    specviz_helper.load_spectrum(spectrum1d, data_label="data1")
    specviz_helper.load_spectrum(spectrum1d, data_label="data2")
    specviz_helper.app.vue_data_item_remove({'item_name': 'data1'})
    specviz_helper.app.get_viewer("spectrum-viewer").apply_roi(XRangeROI(7000, 7100))
    with pytest.raises(RuntimeError, match="Cannot remove all base data"):
        specviz_helper.app.vue_data_item_remove({'item_name': 'data2'})
    specviz_helper.load_spectrum(spectrum1d, data_label="data3")
    specviz_helper.app.vue_data_item_remove({'item_name': 'data2'})
```